snappass/main.py

```diff
@@ -23,8 +22,12 @@
 }
 
 
+def _id():
+    return uuid.uuid4().hex
+
+
 def set_password(password, ttl):
-    key = id_()
+    key = _id()
```

Comment:
Is `_id()` used (or mocked) elsewhere? If not, it seems like we could just call `uuid.uuid4().hex` directly here.